azure-quantum/azure/quantum/target/pasqal/target.py

```diff
@@ -103,15 +106,26 @@ def submit(
         :type input_data: Any
         :param name: Job name
         :type name: str
+        :param shots: Number of shots, defaults to None
+        :type shots: int
         :param input_params: Input parameters, see :class:`azure.quantum.target.pasqal.InputParams` for details.
         :type input_params: Union[InputParams, None, Dict[str, Any]]
         :return: Azure Quantum job
         :rtype: Job
         """
+
         if isinstance(input_params, InputParams):
             typed_input_params = input_params
             input_params = {
-                "runs": typed_input_params.runs,
+                self.__class__._SHOTS_PARAM_NAME: typed_input_params.runs,
```

Comment:
this used to be `runs` earlier and right now we seem to be setting it to `_SHOTS_PARAM_NAME = "count"` on line 67. Is this correct?

Reply:
It seems so. As I noted here https://github.com/microsoft/azure-quantum-python/pull/542#discussion_r1446641584, provider does not respond on `runs` option anymore, and when job is submitted, it falls back to Pascal's default values and returns "count" in details of a result job.